Tasks/ShellScript/task.json

```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
```

Comment:
should be specifyWorkingDirectory default to false and set the visibility of the Working Directory field to be hidden if this value is false.